poetry/console/commands/new.py

```diff
@@ -16,6 +16,7 @@ class NewCommand(Command):
     arguments = [argument("path", "The path to create the project at.")]
     options = [
         option("name", None, "Set the resulting package name.", flag=False),
+        option("author", None, "Author name of the package.", flag=False),
```

Comment:
I am wondering if we should use `--author-name/--author-email`? Would also be great to add the default value in as well.

Reply:
>I am wondering if we should use `--author-name/--author-email`?

Would that be *instead* of `--author`, or *with* `--author`? Should these options then be implemented on the `init` command as well?

>Would also be great to add the default value in as well.

Would the default value be git config user fields if available, `None` if not available? This is already the behavior on line 60, though not explicitly defined in the `option` definition.

Reply:
I would think instead of. 

As for the defaults, yes, same behaviour but initialised outside of the command so that it can be shown in the help message.